Dockerfile

```diff
@@ -27,8 +39,8 @@ RUN cd /tmp \
 RUN mkdir -p /code
 WORKDIR /code
 
-COPY ./package.json /code/package.json
-RUN npm install
+COPY ./package.json ./yarn.lock /code/
```

Comment:
@icereval explained this to me, the intermediate containers get built after each command, so to keep them as small as possible, it is necessary to separate them out.